src/InterestPayout.Common/Persistence/DatabaseContext.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace InterestPayout.Common.Persistence
+{
+    public class DatabaseContext : DbContext
+    {
+        public static string SchemaName { get; } = "interest_payout";
+        public static string MigrationHistoryTable { get; } = HistoryRepository.DefaultTableName;
+
+        public DatabaseContext(DbContextOptions<DatabaseContext> options)
+            : base(options)
+        {
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            base.OnConfiguring(optionsBuilder);
+
+            optionsBuilder.UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking);
```

Comment:
I did that, but please make note of the fact that I had to install additional package https://github.com/efcore/EFCore.NamingConventions since there is no built-in functionality that supports snake case in EF core.